docs/modules/servers/pages/postgres/operate/webadmin/admin-mail-queues-extend.adoc

```diff
@@ -0,0 +1 @@
+// The document only covers Postgres
```

Comment:
Only cover Cassandra/Distributed to be more correct?

Reply:
My mean, this file is meant to cover Postgres documentation but is currently empty.